#!/usr/bin/env python
PACKAGE = "robot_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Kp",						double_t,  	0, "An double parameter", 	1.0,  0.0, 30)
gen.add("Ki",						double_t,  	0, "An double parameter", 	15.0,  0.0, 30)
gen.add("Kd",						double_t,  	0, "An double parameter", 	0.05,  0.0, 5)
gen.add("K1",						double_t,  	0, "An double parameter", 	1.0,  0.0, 5)
gen.add("K2",						double_t,  	0, "An double parameter", 	1.3,  0.0, 5)
gen.add("K3",						double_t,  	0, "An double parameter", 	1.3,  0.0, 5)
gen.add("prozentAngVel",						double_t,  	0, "An double parameter", 	0.03,  0.0, 600)
gen.add("prozentLinVel",						double_t,  	0, "An double parameter", 	0.03,  0.0, 600)
gen.add("robotSpeed",			double_t, 	0, "An double parameter", 	0.2,  0.0, 0.5)
gen.add("robotStartStop",		bool_t,		0, "An boolean parameter", 	False)
exit(gen.generate(PACKAGE, "robot_navigation", "TrackingControl"))
